src/backlight/portfolio/portfolio.py

```diff
@@ -90,14 +90,99 @@ def construct_portfolio(
     return Portfolio(positions)
 
 
-def calculate_pl(pt: Portfolio, mkt: List[MarketData]) -> pd.DataFrame:
+# def calculate_pl(pt: Portfolio, mkt: List[MarketData]) -> pd.DataFrame:
+#     """
+#     Apply the sum on the homogenized portfolio
+#     args:
+#         - portfolio : a defined portfolio
+#         - mkt : list of marketdata for each asset
+#         - base_ccy : asset of reference
+
+#     """
+#     pl = pt.value()
+#     return pl.sum(axis=1)
+
+
+def homogenize_pl(
+    pt: Portfolio, mkt: List[MarketData], base_ccy: str = "USD"
```

Comment:
For `base_ccy: str`, we need to define Currency type somewhere.

Reply:
Should we create a new class in a new folder or add it in a file, like Portfolio.py?

Reply:
How about creating `backlight/assets/currency.py`?

Reply:
`pt` -> `portfolio`.